### ISSUE TITLE
Remove anchor for buttons dropdowns, as it can confuse screenreader users

### ISSUE THREAD

**Issue by mejackreed** [projectblacklight/blacklight#894]
The “Sort by relevance” and “number per page” options are identified as links for screen reader users, and each is followed by a button that doesn’t have a label. Visually these are drop-down menus. This may be confusing. I was not sure what the unlabeled buttons did, and the visual/auditory experiences are not the same.
